framework/src/modules/chain/helpers/config.js

```diff
@@ -104,7 +104,12 @@ function Config(packageJson, parseCommandLineOptions = true) {
 
 	let customConfig = {};
 	if (program.config || process.env.LISK_CONFIG_FILE) {
-		customConfig = loadJSONFile(program.config || process.env.LISK_CONFIG_FILE);
+		const customConfigPath = path.join(
+			__dirname,
+			'../../../../../',
+			program.config || process.env.LISK_CONFIG_FILE
```

Comment:
What happens if I pass an absolute path to `-c`?

Reply:
Do we want that behavior? Until now, the path was relative from root of project. @fchavant

Reply:
OK that's fine then.

Reply:
Decided to keep the relative path for now.